### PR TITLE
add aws camels data

### PR DESCRIPTION
**Pull request by jhamman**

### CODE REVIEW

File: intake-catalogs/hydro/camels.yaml [hunk 1]
```diff
@@ -19,8 +19,29 @@ sources:
...
+      urlpath: 's3://pangeo-camels/basin_timeseries_v1p2_metForcing_obsFlow/basin_dataset_public_v1p2/basin_mean_forcing/{{ source }}/*/{{ basin_id }}_*.txt'
+      csv_kwargs:
+        sep: '\s+'
+        header: 3
+        parse_dates: {'date': ['Year', 'Mnth', 'Day', 'Hr']}
+    metadata:
+      origin_url: 'https://ral.ucar.edu/solutions/products/camels'
 
-  usgs_streamflow:
+  usgs_streamflow_gcp:
```
Comment (rabernat) [inline intake-catalogs/hydro/camels.yaml]: Should we worry about breaking user code when we rename things? Can we have aliases?

**Comment by rabernat** [comment]
Don't know what camels is, but I love data! 🐫

File: intake-catalogs/hydro/camels.yaml (hunk 1 above)
Comment (jhamman) [reply intake-catalogs/hydro/camels.yaml]: I don't think that is necessary just yet. These existing entries just went in so I doubt there are any users at this point (except for me).

**Comment by jhamman** [comment]
> Don't know what camels is, but I love data! 🐫

CAMELS: CATCHMENT ATTRIBUTES AND METEOROLOGY FOR LARGE-SAMPLE STUDIES

**Pull request merged**